copyright.txt

```diff
@@ -1,9 +1,5 @@
 ﻿Some code adapted from http://code.makery.ch/library/javafx-8-tutorial/ by Marco Jakob
 
```

Comment:
Can we add a copyright acknowledgement for the FoodRem chicken icon?

Reply:
![image](https://user-images.githubusercontent.com/78332456/197369745-73aafc4e-cc6a-4932-838b-fe8892944219.png)

Reply:
![image](https://user-images.githubusercontent.com/78332456/197369765-6ad696b5-a288-48c4-8cde-6cce5c11824d.png)

Reply:
Im not sure if I understand it correctly but I do not think figma has copyright

Reply:
Creative Commons **Atribution** 4.0 means you can freely adapt as long as you give credit (attribution) to the original creator. [More Info](https://smartcopying.edu.au/quick-guide-to-creative-commons/).